api/src/main/java/org/eclipse/microprofile/metrics/Timer.java

```diff
@@ -29,7 +29,7 @@
 /**
  * A timer metric which aggregates timing durations and provides duration statistics
  *
- * The timer measures duration in nanoseconds.
+ * The timer measures duration in seconds.
  */
 public interface Timer extends Metric, Sampling {
     /**
```

Comment:
However,

The Context (or in micrometer's context, a `Sample`) when calling `stop` would return long (for nanoseconds)
```    
interface Context extends Closeable {
        /**
         * Updates the timer with the difference between current and start time. Call to this method will not reset the
         * start time. Multiple calls result in multiple updates.
         * 
         * @return the elapsed time in nanoseconds
         */
        long stop();
        /** Equivalent to calling {@link #stop()}. */
        @Override
        void close();
    }
```

Reply:
What does it mean if someone specifies that the unit should be microseconds in an annotation?  Does that affect the value that you get from the Snapshot?

Reply:
See @donbourne  comment below